src/docfx/localization/LocalizationConvention.cs

```diff
@@ -50,12 +48,15 @@ public static (string remote, string branch) GetLocalizationRepo(LocalizationMap
 
             var newLocale = mapping == LocalizationMapping.Repository ? $".{locale}" : ".localization";
             var newBranch = bilingual ? $"{branch}-sxs" : branch;
-            var repoName = remote.Split(new char[] { '/', '\\' }).Last();
-            var match = s_repoNameWithLocale.Match(repoName);
-            if (match.Success && match.Groups.Count >= 2 && !string.IsNullOrEmpty(match.Groups[1].Value))
+
+            if (remote.EndsWith($".{defaultLocale}", StringComparison.OrdinalIgnoreCase))
```

Comment:
extract this method to a method? duplicated in `GetLocalizationTheme`. Might be more duplication